dev/bots/test.dart

```diff
@@ -1645,6 +1648,313 @@ Future<void> _runDocs() async {
   );
 }
 
+// Verifies binaries are codesigned.
+Future<void> _runVerifyCodesigned() async {
```

Comment:
I think if we port this here, we'd want unit tests of this logic. We test the existing code here: https://github.com/flutter/flutter/blob/master/dev/conductor/core/test/codesign_test.dart#L261-L529

Reply:
Migrated the tests. This will help to deprecate adhoc tests while the codesign + entitlements validation is migrated to the codesign tool.